components/gitpod-db/src/typeorm/team-db-impl.ts

```diff
@@ -79,6 +108,14 @@ export class TeamDBImpl implements TeamDB {
             throw new Error('Please choose a team name containing only letters, numbers, -, _, \', or spaces.');
         }
         const slug = name.toLocaleLowerCase().replace(/[ ']/g, '-');
+        if (FORBIDDEN_SLUGS.indexOf(slug) !== -1) {
+            throw new Error('Creating a team with this name is not allowed');
```

Comment:
**question**: This is probably out of the scope of these changes but could we skip the first part of the error message here as it does not seem much helpful for the user? What do you think? Happy to move this discussion to a new follow up issue. Maybe there're cases where the first part is helpful. 🤷 

| BEFORE | AFTER |
|-|-|
| Request createTeam failed with message: Creating a team with this name is not allowed | Creating a team with this name is not allowed |